Consul.Test/Properties/AssemblyInfo.cs

```diff
@@ -34,5 +34,9 @@
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
 
-// This is to avoid using too many concurrent http connections with consul agent
-[assembly: CollectionBehavior(MaxParallelThreads = 20)]
```

Comment:
`MaxParallelThreads` doesn't actually reduce the number of tests that can be run in parallel. It only sets the limit on the number of threads in the thread pool (which doesn't really affect parallelism of asynchronous code).